packages/drivers/odsp-driver/src/test/createNewUtilsTests.spec.ts

```diff
@@ -204,4 +195,39 @@ describe("Create New Utils Tests", () => {
         });
         await epochTracker.removeEntries().catch(() => {});
     });
+
+    it("Should set the isClpCompliantApp prop on resolved url", async () => {
+        const hashedDocumentId = await getHashedDocumentId(driveId, itemId);
+        const localCache = createUtLocalCache();
+        // use null logger here as we expect errors
+        const epochTracker = new EpochTracker(
+            localCache,
+            {
+                docId: hashedDocumentId,
+                resolvedUrl,
+            },
+            new TelemetryNullLogger());
+
+        const fileEntry: IFileEntry = {
+            docId: hashedDocumentId,
+            resolvedUrl,
+        };
+
+        const odspResolvedUrl = await mockFetchOk(
+                async () => createNewFluidFile(
+                    async (_options) => "token",
+                    newFileParams,
+                    new TelemetryNullLogger(),
+                    createSummary(),
+                    epochTracker,
+                    fileEntry,
+                    true,
+                    false,
+                    true,
```

Comment:
```suggestion
                    true /* isClpCompliantApp */,
```

Reply:
And also explicitly validate that the value is false if that param is missing or `false`? (maybe in an existing test that doesn't set it)

Reply:
done. Added in same test.